src/styles/utils/_varibles.scss

```
@@ -0,0 +1,15 @@
+$calendar-gap-size: 1px;
+$cell-size: 100px;
+$calendarWidth: (7 * $calendar-gap-size) + (7 * $cell-size);
+$weekDay: 
+  "mon" 0,
+  "tue" $cell-size + $calendar-gap-size,
+  "wed" $cell-size * 2 + $calendar-gap-size * 2,
```

You can calc it just one time, instead calc each time
```suggestion
$calendarWidth: (7 * $calendar-gap-size) + (7 * $cell-size);
$dayMargin:  $cell-size + $calendar-gap-size;
$weekDay: 
  "mon" 0,
  "tue" $dayMargin,
  "wed" $dayMargin * 2,
```